### DISCUSSION TITLE
Welcome to HxCFloppyEmulator Discussions!

### DISCUSSION THREAD

**Discussion by jfdelnero** [jfdelnero/HxCFloppyEmulator#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WetNoodle** [comment]
I'm trying to find a way to format a USB drive to look like a floppy disk. My dad bought an OLD CNC machine, and to load a program it required a floppy disk. The previous owner converted it to a USB drive but it can still only read the FAT12 file format. Will this work for that? So far this is the closest thing I can find to what I'm looking for.
